src/Neo.CLI/config.fs.mainnet.json

```diff
@@ -24,7 +24,8 @@
       "NeoNameService": "0x7061fbd31562664b58f422c3dee4acfd70dba8af"
     },
     "Plugins": {
-      "DownloadUrl": "https://api.github.com/repos/neo-project/neo-modules/releases"
+      "DownloadUrl": "https://api.github.com/repos/neo-project/neo/releases",
+      "CustomUrls": []
```

Comment:
I think that the `CustomUrls` is not so important in my opinion, the important thing now is to show the installed plugins correctly, if they are custom.

Reply:
They can just change this download URL, and release on their `github` already.